nemo_skills/training/nemo_rl/start_sft.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         output_key: str = "output",
         num_proc: int | None = None,
         force_reprocess: bool = False,
+        input_template_path: str | None = None,
```

Comment:
also wouldn't this fail for typical cases when e.g. you have keys like problem and generation in your file, but the logic for setting defaults checks for input / output or messages? So that part should also be updated

Reply:
@Kipok I didn't understand this. When a user is passing `input_template_path`, their provided `input_key` field must match with keys present in the `input_template`. Why having keys like problem and generation would cause failure?